src/main/java/io/seqera/tower/cli/responses/TaskView.java

```diff
@@ -0,0 +1,151 @@
+
+import io.seqera.tower.cli.utils.FormatHelper;
+import io.seqera.tower.cli.utils.TableList;
+
+import java.io.PrintWriter;
+import java.time.OffsetDateTime;
+import java.util.HashMap;
+import java.util.Map;
+
+public class TaskView extends Response {
+
+    public final Map<String, Object> general;
+    public final String command;
+    public final String environment;
+    public final Map<String, Object> times;
+    public final Map<String, Object> resources;
+    public final Map<String, Object> usage;
+
+    public TaskView(
+            Map<String, Object> general,
+            String command,
+            String environment,
+            Map<String, Object> times,
+            Map<String, Object> resources,
+            Map<String, Object> usage
+    ) {
+        this.general = general;
+        this.command = command;
+        this.environment = environment;
+        this.times = times;
+        this.resources = resources;
+        this.usage = usage;
+    }
+
+    @Override
+    public Object getJSON() {
+        Map<String, Object> data = new HashMap<>();
+
+        data.put("general", general);
+
+        if (command != null) {
+            data.put("command", command);
+        }
+
+        if (environment != null) {
+            data.put("environment", environment);
+        }
+
+        if (!times.isEmpty()) {
+            data.put("times", times);
+        }
+
+        if (!resources.isEmpty()) {
+            data.put("resources", resources);
+        }
+
+        if (!usage.isEmpty()) {
+            data.put("usage", usage);
+        }
+
+        return data;
+    }
+
+    @Override
+    public void toString(PrintWriter out) {
+        out.println(ansi(String.format("%n%n    @|bold  %s|@", "General")));
+        TableList tableGeneral = new TableList(out, 2);
+        tableGeneral.setPrefix("    ");
+        tableGeneral.addRow("Task ID", general.get("taskId").toString());
+        tableGeneral.addRow("ID", general.get("id").toString());
+        tableGeneral.addRow("Work directory", general.get("workDir").toString());
+        tableGeneral.addRow("Status", general.get("status").toString());
+        tableGeneral.print();
+
+        if (command != null) {
+            out.println(ansi(String.format("%n     @|bold Command|@")));
+            out.println(ansi("    ---------"));
+            out.println(ansi(String.format("     %s", command.trim().replaceAll("\n", String.format("%n     ")))));
+        }
+
+        if (environment != null) {
+            out.println(ansi(String.format("%n     @|bold Environment|@")));
+            out.println(ansi("    -------------"));
+            out.println(ansi(String.format("     %s", environment.trim().replaceAll("\n", String.format("%n     ")))));
+        }
+
+        if (!times.isEmpty()) {
+            out.println(ansi(String.format("%n%n    @|bold  %s|@%n", "Execution time")));
+            TableList tableTime = new TableList(out, 3, "Label", "Value", "Description");
```

Comment:
I think that to follow same formatting as in other sections, even if these sections have three columns, there is no need to set the headers because the content is self-explanatory. I'd go to something more like this:
```
     Execution time
    ----------+-------------------------------+-----------------------------------------------------------------------------------------------
     submit   | Thu, 28 Oct 2021 11:12:08 GMT | Timestamp when the task has been submitted                                                    
     start    | Thu, 28 Oct 2021 11:14:07 GMT | Timestamp when the task execution has started                                                 
     complete | Thu, 28 Oct 2021 11:14:27 GMT | Timestamp when task execution has completed                                                   
     duration | 18s 2m                        | Time elapsed to complete since the submission i.e. including scheduling time                  
     realtime |                               | Task execution time i.e. delta between completion and start timestamp i.e. compute wall-time
```